ai21/models/chat/chat_message.py

```diff
@@ -2,9 +2,7 @@
 
 from typing_extensions import TypedDict, Literal
 
-from .role_type import RoleType
-
 
 class ChatMessage(TypedDict):
-    role: RoleType | Literal["user", "assistant"]
+    role: Literal["user", "assistant"]
```

Comment:
You might want to add "system" as well - https://www.notion.so/ai21labs/AI21-Foundation-Models-API-v2-d39ae6f803e24266809c82d5bb9db00d?pvs=4#2148c84cf696426180a52fb23d046a44

Reply:
I actually removed this type entirely and changed it to `str` as neither the vllm and the server validate these enum/literal

Reply:
@tomeras91 WDYT? do we want to validate the role?
@Josephasafg even if we don't validate it, maybe you should keep the Literal for ease of use alone.

Reply:
The only caveat here is that we would need to maintain the roles in the SDK if anything changes (more/less are added/removed). 
We could either use a Literal for ease of use and documentation or just add a docstring explaining what is supported if there is no agreement. 

@Asaf-m @tomeras91 why wasn't it enforced in the server and in the serving?

Reply:
We should validate the roles in vllm-serving. I.e. - the model server maintained by the Serving team
I can't tag Avsha for some reason